README.rst

```diff
@@ -28,6 +28,7 @@ Requirements
 ============
 This branch requires 3.1.3 version MU1 or later of the HPE 3PAR firmware
 This branch requires 4.0.0 version of the HPE Primera firmware
+This branch requires 9.4.0 version of the HPE Alletra firmware
```

Comment:
Change the version to 9.3.0 instead of 9.4.0. Also add 9000 after Alletra

Reply:
Addressed review comments